docs/modules/ROOT/pages/admin-api-reference.adoc

```diff
@@ -179,6 +179,53 @@ The same call can be made using `defender-admin-client` passing the contract and
 await client.getProposal(contractId, proposalId);
 ```
 
+[[simulate-endpoint]]
+=== Simulate a Proposal
+
+Proposals can be simulated via the API. The results of a simulation (`SimulationResponse`) will be stored and can be retrieved with the `getProposalSimulation` endpoint.
+
+```js
+const proposal = await client.getProposal(contractId, proposalId);
+
+// import the ABI and create an ethers interface
+const contractInterface = new utils.Interface(contractABI);
+
+// encode function data
+const data = contractInterface.encodeFunctionData(proposal.functionInterface.name, proposal.functionInputs);
+
+const simulation = await client.simulateProposal(
+  proposal.contractId, // contractId
+  proposal.proposalId, // proposalId
+  {
+    transactionData: {
+      // this is the default hardhat address
+      from: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266', // change this to impersonate the `from` address
+      type: 'function-call', // or 'send-funds'
```

Comment:
Is this actually used? I looked at the code and I don't see it's relevant anywhere.
We can keep it, it just drew my attention

Reply:
Good catch! Indeed, doesn't look like we use it anywhere..